Core/Security/VOMS.py

```diff
@@ -193,10 +193,11 @@ def getVOMSESLocation(self):
     requiredFilePerms = stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH
     # 777
     allPerms = stat.S_IRWXU | stat.S_IRWXG | stat.S_IRWXO
-    vomsesPaths = []
-    if 'DIRAC_VOMSES' in os.environ:
-      vomsesPaths.append(os.environ['DIRAC_VOMSES'])
-    vomsesPaths.append(os.path.join(rootPath, "etc", "grid-security", "vomses"))
```

Comment:
This folder is usually filled when the dirac-configure script is called. And if this default filled vomses is removed then X509_VOMSES _must_ be set in the bashrc file (not just commented as in #4932 at the moment of typing this)

Reply:
#4932 has been updated

Reply:
I think this default directory shouldn't be removed here (at least not as a patch for v7r0), because it would require everyone to update their bashrc files. And If I understand correctly the DIRAC way of updating servers, it would use the current dirac-install of the installation, and would thus not get the updated variable. Though that is also an argument against renaming the env variable like this, because other people might use that already.

Reply:
I would remove this line

https://github.com/DIRACGrid/DIRAC/blob/ea0facf13c75419772c115338280771b042b401b/Core/Security/VOMS.py#L202

and add one that looks for `X509_VOMSES`. And return an error (with minimal instructions) if the required variables are not set.

_Originally posted by @fstagni in https://github.com/DIRACGrid/DIRAC/issues/4846#issuecomment-746250833_

I followed what was agreed, but you have a valid point that this is a quite major intervention for a patch release.

Reply:
@fstagni @chaen any thoughts

Reply:
I agree with @andresailer , so be less "aggressive" here.

Reply:
I mean, we can drop `DIRAC_VOMSES` in v7r2, but not for patch.

Reply:
OK, I have added some transition code with warning to users that they should adopt to the new behaviour.